docs/source/UserGuide/GettingStarted/InstallingClient/index.rst

```diff
@@ -24,6 +24,7 @@ and now DIRAC::
   $ pip install DIRAC
 
 will install the most recent production version found on https://pypi.org/project/DIRAC/
+Note: DIRACCommon provides core utilities and must be installed alongside DIRAC.
```

Comment:
Is this comment needed? While technically correct, I do not think that a possible user, installing DIRAC, needs to know about it.